webpack.config.js

```diff
@@ -22,35 +22,69 @@ module.exports = {
         path: __dirname + "/dist/",
     },
 
-
     // file resolutions
     resolve: {
-        extensions: [".ts", ".js",".tsx",".jsx"]
+        extensions: [".ts", ".js", ".tsx", ".jsx"],
     },
 
     // loaders
     module: {
         rules: [
             {
                 test: /\.tsx?$/,
-                use: 'ts-loader',
+                use: "ts-loader",
                 exclude: /(node_modules)|(ts\/types)/,
             },
             { test: /\.d.ts$/, use: "raw-loader" },
+            {
+                test: /\.css$/,
+                exclude: /node_modules/,
+                use: [
+                    {
+                        loader: "style-loader",
+                    },
+                    {
+                        loader: "css-loader",
+                        options: {
+                            importLoaders: 1,
+                        },
+                    },
+                    {
+                        loader: "postcss-loader",
+                        options: {
+                            postcssOptions: {
+                                plugins: [
+                                    require("postcss-import"),
+                                    require("postcss-prefix-selector")({
+                                        // that's for tailwind not to mess with bootstrap styles
+                                        prefix: ".addon",
+                                        transform: function (prefix, selector, prefixedSelector) {
+                                            if (selector.startsWith("html") || selector.startsWith("body")) {
+                                                return selector;
+                                            }
+                                            return prefixedSelector;
+                                        },
+                                    }),
+                                    require("tailwindcss"),
+                                    require("autoprefixer"),
+                                ],
+                            },
```

Comment:
can we utilise `postcss.config.js` here?

Reply:
yeah, we can and we should, but magically it works only when I duplicate that thing 😅 Removing postcss config or changing `postcss-prefix-selector` in webpack just break it. I'll investigate later